compiler-core/src/type_/lsp_pretty.rs

```diff
@@ -0,0 +1,113 @@
+use std::sync::Arc;
+
+use ecow::EcoString;
+use im::HashMap;
+use itertools::Itertools;
+
+use crate::{
+    //ast::TypeAst,
+    pretty::{break_, concat, nil, Document, Documentable},
+};
+
+use super::{pretty::Printer, Type, TypeVar};
```

Comment:
I think we can just append to a string as inline hints don't need to be wrapped onto multiple lines. Would make it simpler and faster

Reply:
Ah gotcha ! Will change this.